src/templates/ObservabilityPackDetails.js

```diff
@@ -37,6 +37,169 @@ const allowedElements = [
+        commodo consequat. Duis aute irure dolor in reprehenderit in voluptate
+        velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint
+        occaecat cupidatat non proident, sunt in culpa qui officia deserunt
+        mollit anim id est laborum.
+      </Intro>
+      {alertContent}
+    </>
+  );
+};
+
+const renderSynthetics = () => {
+  return (
+    <Intro
+      css={css`
+        margin-bottom: 16px;
+      `}
+    >
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+      tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
+      veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
+      commodo consequat. Duis aute irure dolor in reprehenderit in voluptate
+      velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat
+      cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id
+      est laborum.
+    </Intro>
+  );
+};
+
+const renderVisualizations = () => {
+  return (
+    <Intro
+      css={css`
+        margin-bottom: 16px;
+      `}
+    >
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+      tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
+      veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
+      commodo consequat. Duis aute irure dolor in reprehenderit in voluptate
+      velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat
+      cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id
+      est laborum.
+    </Intro>
+  );
+};
+
+const emptyStateContent = (pack, tabName) => {
+  return (
+    <div
+      css={css`
+        border: 1px solid var(--divider-color);
+        border-radius: 0.25rem;
+        padding: 1rem;
+      `}
+    >
+      <div
+        css={css`
+          display: flex;
+          justify-content: center;
+          padding: 2rem;
+        `}
+      >
+        <Icon
+          css={css`
+            font-size: 4rem;
+            color: var(--divider-color);
+          `}
+          name="edit"
+        />
+      </div>
+      <p
+        css={css`
+          text-align: center;
+        `}
+      >
+        This pack doesn't include any {tabName}. Do you think it should?
+        <br />
+        You can modify this pack to add useful components. View the pack source
+        code and open a pull request.
+      </p>
+      <div
+        css={css`
+          display: flex;
+          justify-content: center;
+        `}
+      >
+        <Button
+          as={Link}
+          variant={Button.VARIANT.PRIMARY}
+          to={getPackUrl(pack.logoUrl)}
+          rel="noopener noreferrer"
+          instrumentation={{ packName: pack.name }}
+        >
+          <Icon
+            name="fe-github"
+            css={css`
+              margin-right: 7px;
+            `}
+          />
+          View Repo
+        </Button>
+      </div>
```

Comment:
This may be a nit or not, but the double `View Repo` button looks a little off
<img width="326" alt="Screenshot 2021-07-12 at 15 10 33" src="https://user-images.githubusercontent.com/17122543/125303339-a286c080-e324-11eb-859a-c07c4f3783f3.png">

Reply:
I agree that it's a bit odd to have the double button, especially on mobile. @jpvajda Do you have any thoughts on this? I think we can leave it for mobile nits